pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -120,6 +168,22 @@ func toRuntimeProtocol(protocol api.Protocol) runtimeApi.Protocol {
 	return runtimeApi.Protocol_TCP
 }
 
+// toKubeContainer converts runtimeApi.Container to kubecontainer.Container.
+func (m *kubeGenericRuntimeManager) toKubeContainer(c *runtimeApi.Container) (*kubecontainer.Container, error) {
+	if c == nil || c.Id == nil || c.Name == nil || c.Image == nil || c.State == nil {
```

Comment:
@yifan-gu Right. All fields in protobuf are set optional because that will be easy for upgrading to proto3. (in proto3, all fields are optional by default)